JetNews/app/src/main/java/com/example/jetnews/ui/article/ArticleScreen.kt

```diff
@@ -213,7 +213,7 @@ private fun BottomBar(
         Row(
             verticalAlignment = Alignment.CenterVertically,
             modifier = Modifier
-                .height(56.dp)
+                .height(102.dp)
```

Comment:
This adds an extra fixed height of `46dp`, but we want that height to based on the insets padding applied below.

The more flexible solution here is to swap the order of the `windowInsetsPadding` and the `height(56.dp)`, so that we set the height after the `windowInsetsPadding` is applied, which ensures that the insets padding doesn't take away the space available to the buttons.

Reply:
Ah I see, yes that makes a lot more sense. Will update that now, thanks for the explanation